test/IDE/complete_rdar147789214.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %batch-code-completion
+
+// rdar://147789214 - Make sure we insert generic parameters for underlying type.
+
+struct S<T> {}
+typealias Foo = S
+typealias Invalid = (S, S)
+
+let _: S = #^COMPLETE^#
+// COMPLETE-DAG: Decl[TypeAlias]/CurrModule: Foo[#S<T>#]; name=Foo
```

Comment:
Is this different than what you'd get if you wrote `typealias Foo<T> = S<T>`?

Reply:
I don't believe so, no